packages/services/api/src/modules/audit-logs/resolvers/CollectionUpdatedAuditLog.ts

```diff
@@ -0,0 +1,27 @@
+import type { CollectionUpdatedAuditLogResolvers } from './../../../__generated__/types';
+
+/*
+ * Note: This object type is generated because "CollectionUpdatedAuditLogMapper" is declared. This is to ensure runtime safety.
+ *
+ * When a mapper is used, it is possible to hit runtime errors in some scenarios:
+ * - given a field name, the schema type's field type does not match mapper's field type
+ * - or a schema type's field does not exist in the mapper's fields
+ *
+ * If you want to skip this file generation, remove the mapper or update the pattern in the `resolverGeneration.object` config.
+ */
+export const CollectionUpdatedAuditLog: CollectionUpdatedAuditLogResolvers = {
+  __isTypeOf: e => e.event_action === 'COLLECTION_UPDATED',
+  eventTime: e => new Date(e.timestamp).toISOString(),
+  collectionId: e => e.metadata.collectionId,
+  updatedFields: e => e.metadata.updatedFields,
+  collectionName: e => e.metadata.collectionName,
+  id: e => e.id,
+  record: async (event, _arg, _ctx) => {
+    return {
+      userEmail: event.user_email,
+      userId: event.user_id,
+      organizationId: event.organization_id,
+      user: event.metadata.user,
+    };
+  },
```

Comment:
@TuvalSimha all `record` resolvers are identical? If so, maybe we should pass 
```
{
  eventTime,
  collectionId, 
  ..., 
  record: { userEmail...} 
}
```

to a resolver resolving those AuditLog types?

This way you won't repeat yourself 100x times and when we make changes, you do 1 instead of 100.

Reply:
what I mean here is that when you pass data to `Organization.auditLogs` or something, this data is already transformed and has `record` object in it, same with `eventTime` and other shared fields.

Reply:
And because we have typescript and strict type definitions, it makes sure we have `record` available and in correct shape, so no need to repeat the same logic 100 times.